assets/sass/components/_personalised.scss

```diff
@@ -85,6 +81,11 @@
   background-color: govuk-colour('purple');
   margin-right: 0;
 }
+.iep-level--placeholder {
+  color: govuk-colour('black');
+  background-color: rgba(177, 180, 182, 0.5);
```

Comment:
Does this need opacity or is it meant to be a solid colour?

Reply:
Just to be consistent with the balance placeholders IIRC

Reply:
OK 👍